specification/support/resource-manager/Microsoft.Support/preview/2021-06-01-preview/supportResourceIdDetails.json

```diff
@@ -0,0 +1,291 @@
+    "Operation": {
+      "description": "The operation supported by Microsoft Support resource provider.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}.",
+          "type": "string",
+          "readOnly": true
+        },
+        "display": {
+          "description": "The object that describes the operation.",
+          "properties": {
+            "description": {
+              "description": "The description of the operation.",
+              "type": "string",
+              "readOnly": true
+            },
+            "operation": {
+              "description": "The action that users can perform, based on their permission level.",
+              "type": "string",
+              "readOnly": true
+            },
+            "provider": {
+              "description": "Service provider: Microsoft Support.",
+              "type": "string",
+              "readOnly": true
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed.",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "LookUpResourceIdRequest": {
+      "description": "The look up resource Id request body",
+      "type": "object",
+      "properties": {
+        "identifier": {
+          "description": "The System generated Id that is unique. Use supportTicketId property for Microsoft.Support/supportTickets resource type.",
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Support/supportTickets"
+          ],
+          "x-ms-enum": {
+            "name": "Type",
+            "modelAsString": false
+          },
+          "description": "The type of resource."
+        }
+      }
+    },
+    "LookUpResourceIdResponse": {
+      "description": "The look up resource id response",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "The resource Id of support resource type.",
+          "type": "string"
+        }
+      }
+    },
+    "ExceptionResponse": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ServiceError",
+          "description": "The API error details."
+        }
+      },
+      "description": "The API error."
+    },
+    "ServiceError": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServiceErrorDetail"
```

Comment:
>ServiceErrorDetail [](http://example.com/codeflow?start=35&length=18)

The ARM error definition is completely recursive, so this could be a reference to `ServiceError`. In general, you should avoid declaring your own definitions for common contracts and reference them out of common types v2 area of the repo. (This comment doesn't block the PR, though.)

Reply:
Removed error code from manifest and example